lib/collectinfo_analyzer/get_controller.py

```diff
@@ -308,3 +304,44 @@ def get_xdr_namespaces(self, for_mods=None):
                             del ns_stats[ns]
 
         return stats
+
+
+class GetAclController:
```

Comment:
I am trying to move everything over to using a GetController to match live_cluster mode